numerics/elliptic_integrals.cpp

```diff
@@ -1435,158 +1429,163 @@ void FukushimaEllipticBsDsMaclaurinSeries(double const y,
+                          fukushima_elliptic_Js_maclaurin_n_1_1.Evaluate(n)));
   PolynomialInMonomialBasis<double, double, 2, HornerEvaluator>
-      fukushima_elliptic_js_maclaurin_m_2(
-          std::make_tuple(fukushima_elliptic_js_maclaurin_n_2_0.Evaluate(n),
-                          fukushima_elliptic_js_maclaurin_n_2_1.Evaluate(n),
-                          fukushima_elliptic_js_maclaurin_n_2_2.Evaluate(n)));
+      fukushima_elliptic_Js_maclaurin_m_2(
+          std::make_tuple(fukushima_elliptic_Js_maclaurin_n_2_0.Evaluate(n),
+                          fukushima_elliptic_Js_maclaurin_n_2_1.Evaluate(n),
+                          fukushima_elliptic_Js_maclaurin_n_2_2.Evaluate(n)));
   PolynomialInMonomialBasis<double, double, 3, HornerEvaluator>
-      fukushima_elliptic_js_maclaurin_m_3(
-          std::make_tuple(fukushima_elliptic_js_maclaurin_n_3_0.Evaluate(n),
-                          fukushima_elliptic_js_maclaurin_n_3_1.Evaluate(n),
-                          fukushima_elliptic_js_maclaurin_n_3_2.Evaluate(n),
-                          fukushima_elliptic_js_maclaurin_n_3_3.Evaluate(n)));
+      fukushima_elliptic_Js_maclaurin_m_3(
+          std::make_tuple(fukushima_elliptic_Js_maclaurin_n_3_0.Evaluate(n),
+                          fukushima_elliptic_Js_maclaurin_n_3_1.Evaluate(n),
+                          fukushima_elliptic_Js_maclaurin_n_3_2.Evaluate(n),
+                          fukushima_elliptic_Js_maclaurin_n_3_3.Evaluate(n)));
   PolynomialInMonomialBasis<double, double, 4, EstrinEvaluator>
-      fukushima_elliptic_js_maclaurin_m_4(
-          std::make_tuple(fukushima_elliptic_js_maclaurin_n_4_0.Evaluate(n),
-                          fukushima_elliptic_js_maclaurin_n_4_1.Evaluate(n),
-                          fukushima_elliptic_js_maclaurin_n_4_2.Evaluate(n),
-                          fukushima_elliptic_js_maclaurin_n_4_3.Evaluate(n),
-                          fukushima_elliptic_js_maclaurin_n_4_4.Evaluate(n)));
-  double const js0 = fukushima_elliptic_js_maclaurin_m_0.Evaluate(m);
-  double const js1 = fukushima_elliptic_js_maclaurin_m_1.Evaluate(m);
-  double const js2 = fukushima_elliptic_js_maclaurin_m_2.Evaluate(m);
-  double const js3 = fukushima_elliptic_js_maclaurin_m_3.Evaluate(m);
-  double const js4 = fukushima_elliptic_js_maclaurin_m_4.Evaluate(m);
+      fukushima_elliptic_Js_maclaurin_m_4(
+          std::make_tuple(fukushima_elliptic_Js_maclaurin_n_4_0.Evaluate(n),
+                          fukushima_elliptic_Js_maclaurin_n_4_1.Evaluate(n),
+                          fukushima_elliptic_Js_maclaurin_n_4_2.Evaluate(n),
+                          fukushima_elliptic_Js_maclaurin_n_4_3.Evaluate(n),
+                          fukushima_elliptic_Js_maclaurin_n_4_4.Evaluate(n)));
+  // The first five coefficients Jₗ(n|m); the others are computed as needed.
+  // TODO(egg): J₁(n|m) = 1/3; could we make it constexpr?
+  double const J₁ = fukushima_elliptic_Js_maclaurin_m_0.Evaluate(m);
```

Comment:
The change of index is confusing because J1 is for m^0 now.  Could we go back to 0-based?

Reply:
J<sub>1</sub> is for y<sup>1</sup> in the result; this is 0-based, there just is no constant term.
This is the convention used in the paper.

Your indexing led to -1-based indices in the coefficient of the series.